src/util/logger.ts

```diff
@@ -1,4 +1,21 @@
-import { styleText } from 'node:util';
+import chalk from 'chalk';
+
+type ChalkStyle = keyof typeof chalk;
+
+function styleText(styles: ChalkStyle[], text: string): string {
+  let styledText = text;
+
+  // 应用样式
+  styles.forEach((style) => {
+    let styleFunction: any = chalk[style];
+    if (styleFunction instanceof Function) {
+      styledText = styleFunction(styledText);
+    }
+  });
+
+  return styledText;
+}
```

Comment:
We can import the various types from Chalk and use those to construct our style type.

```typescript
import chalk, { BackgroundColorName, ColorName, ForegroundColorName, ModifierName } from "chalk";

type ChalkStyle =
  | ModifierName
  | ForegroundColorName
  | BackgroundColorName
  | ColorName;

function styleText(styles: ChalkStyle[], text: string): string {
  let styledText = text;

  styles.forEach((style) => {
    const styleFunction = chalk[style];

    if (styleFunction instanceof Function) {
      styledText = styleFunction(styledText);
    }
  });

  return styledText;
}
```